### PR TITLE
Last fixes in th AdkMobile project

### PR DESCRIPTION
**Pull request by Zeeerooo**
- Add links with antennas and carries in the gsm observation events reception
- Add links with carrier in the telephony observation events reception
- Change language in several comments
- Added and modified tests

### CODE REVIEW

**Comment by sebablasko** [comment]
in @Zeeerooo we trust
LGTM

File: app/api/events_view.py [hunk 1]
```diff
@@ -180,32 +180,32 @@ def save_cdma_events(events, device, sim, app_version_code):
         total_events += 1
         eventModel = CdmaEvent()
         event['app_version_code'] = app_version_code
-
+        carrier = None
```
Comment (Phfollert) [inline app/api/events_view.py]: Why?

Comment (Zeeerooo) [inline app/api/events_view.py]: Sorry, I used it in a previous commit

**Comment by Phfollert** [comment]
LGTM

**Comment by dmadariaga** [comment]
LGTM

**Pull request merged**